database_setup/aggregated-user-tracking.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE public.user_tracking
+(
+    username text NOT NULL,
+    id text NOT NULL,
+    email text NOT NULL,
+    latest_login timestamp NOT NULL,
+    inactive_email_sent timestamp,
+    revoke_access timestamp,
```

Comment:
Does `revoke_access` show we have revoked access or we want to revoke access? If it's the former then it should be `revoked_access`